google/cloud/bigquery/_pandas_helpers.py

```diff
@@ -129,9 +133,13 @@ def pyarrow_timestamp():
         pyarrow.binary().id: "BYTES",
         pyarrow.string().id: "STRING",  # also alias for pyarrow.utf8()
         pyarrow.decimal128(38, scale=9).id: "NUMERIC",
+    }
+
+    if int(pyarrow.__version__.split(".")[0]) >= 3:
```

Comment:
This should work, but I'd prefer if we use a more explicit version comparison.

For example: https://github.com/googleapis/python-api-core/blob/7271b23afddb032e49e957525704d0cd5bfa4c65/google/api_core/grpc_helpers.py#L47

Reply:
Also, we can add a constant as is done in `google-api-core` to indicate to tests and other logic if the BIGNUMERIC feature is available.

Reply:
Makes sense.

Re: version comparison - I initially wanted to avoid using `pkg_resources.parse_version()`, because it depends on setuptools, but on a second thought we already implicitly require `setuptools` to install the package, thus relying on `pkg_resources` being present is fine.